src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const TRUE_KEYS = Object.values(robot);
+  const TRUE_VALUES = Object.keys(robot);
+  const OBJ = {};
+
+  for (let i = 1; i < TRUE_KEYS.length; i++) {
+    let FIRST_KEY = 0;
+
+    if (TRUE_KEYS[FIRST_KEY] === TRUE_KEYS[i]) {
+      return null;
+    }
+
+    FIRST_KEY++;
+  }
+
+  for (let y = 0; y < TRUE_KEYS.length; y++) {
+    OBJ[TRUE_KEYS[y]] = TRUE_VALUES[y];
+  }
```

Comment:
You can do this much simpler and with less code using for...of loop:
```
for(const key of robot) {
  <your code>
}
```

Reply:
I`ve tried to rewrite it using for...of loop, but faced a problem that robot is not iterable. I solved this but i am not  so sure that it is an appropriate variant.

Reply:
Sorry, I guess, Victor meant `for...in` loop. it is used `for...in` loop for objects.

Reply:
Thanks), I have solved it with both for...of loop (first solution) and for...in loop (second solution)